styles.css

```diff
@@ -50,6 +50,63 @@ html {
   display: none;
 }
 
+/* mobile menu */
+.mobile_m {
+  display: none;
+  flex-direction: column;
+  position: relative;
```

Comment:
- [ Required ] I think it would be better if you use `position: fixed` instead of `relative` so that when a user is scrolling the page the menu is still in focus. I've also noticed that when the position is set to relative the blurred background on the Figma design is not visible, changing it to `fixed` makes the blurred background take effect.

- [ Required ] I think it would be better if you make sure that the `font-size` and `font-weight` in your design of the mobile menu are consistent with those on the Figma design. 

```CSS
.mobile_m {
  display: none;
  flex-direction: column;
  position: fixed;
  background-color: rgba(60, 70, 255, 0.6);
  height: 100vh;
  width: 100%;
  background-blend-mode: lighten;
  top: 0;
  backdrop-filter: blur(10px);
  padding: 2rem;
}

.mobile_menu li a {
  color: var(--white);
  font-size: 32px;
  line-height: 44px;
  font-weight: 600;
}
```

| Your design ( position: relative ) | Changing to ( position: fixed )|
|-------------------------------------------|---------------------------------------|
|![Screenshot from 2023-07-25 01-23-07](https://github.com/MPDADDY/my_portfolio/assets/16392046/bc5a6f53-3287-4bdc-808e-2145933f39cf)|![Screenshot from 2023-07-25 01-22-34](https://github.com/MPDADDY/my_portfolio/assets/16392046/d8c0bee0-0929-4e4c-93dd-3c2bd1c851b1)|

## Figma design
![Screenshot from 2023-07-25 01-26-30](https://github.com/MPDADDY/my_portfolio/assets/16392046/f12166cf-6674-4504-aa23-aa4d45443e6f)